argilla/tests/unit/test_settings/test_settings.py

```diff
@@ -154,6 +154,15 @@ def test_settings_access_by_out_of_range(self):
         with pytest.raises(IndexError):
             _ = settings.fields[10]
 
+    def test_settings_with_modified_default_task_distribution(self):
```

Comment:
@frascuchon this test case seems unrelated to this PR?

We want to test if the code is backward compatible with existing datasets without distributions and ensure we don't overwrite them or get an error, correct?